android/src/main/java/com/reactnativea11y/modules/services/A11yReader.java

```diff
@@ -0,0 +1,103 @@
+package com.reactnativea11y.modules.services;
+
+
+import android.accessibilityservice.AccessibilityServiceInfo;
+import android.content.Context;
+import android.os.Build;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.View;
+import android.view.accessibility.AccessibilityEvent;
+import android.view.accessibility.AccessibilityManager;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.uimanager.IllegalViewOperationException;
+import com.facebook.react.uimanager.UIManagerModule;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class A11yReader {
+  private static final String TALKBACK_SETTING_ACTIVITY_NAME = "com.android.talkback.TalkBackPreferencesActivity";
+
+  private ReactApplicationContext context;
+
+  public A11yReader(ReactApplicationContext context) {
+    this.context = context;
+    this.startListening();
+  }
+
+  private void startListening() {
+    AccessibilityManager accessibilityManager = (AccessibilityManager) context
+      .getSystemService(Context.ACCESSIBILITY_SERVICE);
+    if (accessibilityManager != null) {
+      accessibilityManager.addAccessibilityStateChangeListener(new AccessibilityManager.AccessibilityStateChangeListener() {
```

Comment:
Can be replace with 
accessibilityManager.addAccessibilityStateChangeListener(this::accessibilityChanged);